mozilla/bedrock
# Automatically generate image size versions that fit our agreed pattern

## Summary

Please read #14578 for context.

This changeset eagerly generates the various sizes of image we _might_ use within CMS-managed pages, ready for use (and to avoid the web pod erroring when it can't generate a revision itself.)

Note that the eager generation is done synchronously in the POST request loop right now - this is fast/fine locally, but we will need to move it to an async worker task (via django-rq, which we'll need to add to things separately, before we can finish this work)

- [X] I used an AI to write some of this code: test skeleton, plus some simple regex tuning


## Issue / Bugzilla link

#14578 

## Testing

* `./manage.py createsuperuser` if you need to
* Log in at /cms-admin/
* Upload an image
* Look in bedrock/custom-images/ and see the various sizes have been pre-cut for you.
